### ISSUE TITLE
manhattan plot 'chr' label crowding

### ISSUE THREAD

**Issue by wconnell** [bernatgel/karyoploteR#84]
How can I adjust the 'chr' labels on manhattan plot? The default results in [overlap/crowding](https://bernatgel.github.io/karyoploter_tutorial//Tutorial/PlotManhattan/PlotManhattan.html). I'm not sure where the parameters are hidden. Thank you.

**Comment by VasLem** [comment]
Yes, how is it possible to set the ticks orientation to e.g. 45 degrees?
EDIT: as always, I found the solution after asking: `plotKaryotype(plot.type = 4, srt=45)`

**Comment by bernatgel** [comment]
Glad you found it! If you have any other problem, please ask!

Bernat
